pkgs/development/python-modules/pymetno/default.nix

```diff
@@ -8,15 +8,15 @@
 }:
 
 buildPythonPackage rec {
-  pname = "PyMetno";
-  version = "0.8.2";
+  pname = "pymetno";
```

Comment:
https://github.com/Danielhiversen/pyMetno/blob/master/setup.py#L4

Why the change?

Reply:
pname should match the name in python-packages.

Reply:
The [docs](https://nixos.org/manual/nixpkgs/unstable/#contributing) says we follow [PEP-0503]https://www.python.org/dev/peps/pep-0503/#normalized-names) for the names.